### DISCUSSION TITLE
"Database not available for this location" ?

### DISCUSSION THREAD

**Discussion by kreativkod** [rern/rAudio#748] [Fixed Issues]
When I click album from the main screen I get the following error message. The other buttons work fine. I get the same error message when I click classical in the genre screen, when the other genres work fine.
Any clues as to why?
![Screenshot_20220110-220754](https://user-images.githubusercontent.com/21079205/148922438-b3908667-2684-4e46-8f46-1b413127331c.jpg)

![Screenshot_20220111-110623](https://user-images.githubusercontent.com/21079205/148922707-2e556079-680a-431f-9a62-7a3e616e43f9.jpg)

**Comment by surfninja2** [comment]
make sure the directory where you have your music is available.  

Look for a green dot in the Systems page showing the mount points.  You might need to remount it.  Or  your NAS might have unmounted (or unshared) the directory.  You will need to remount it.

You could try to mount the music folder from a different computer to make sure your NAS is working.  Then remount on the System page (unmount/mount?).

it happens...

**Reply by kreativkod** [reply]
Correct me if I'm wrong but I don't think that is the problem. It's only when browsing by album or by the genre 'classical'  that this happens.

rAudio fetches all files from one and the same NAS mount and it is not unmounted. rAudio plays files well when the library is browsed in another manner. This to me seems to be an issue with browsing the library.

**Comment by ceejayemm** [comment]
I am having a similar, but not quite the same, problem, I am however using the new Shared Data option. I don't think it is exactly the same as you as I don't get the database error that you are getting but I do get an issue where selecting a Genre returns some albums / tracks and other Genres return no albums/tracks.  This seems, for me, to occur on the systems which were NOT used to populate the Shared Data. On the system that was used to populate the Shared Data I seem to get most albums/tracks for most Genres - but I haven't counted them all.

Chris

**Comment by hjfisch** [comment]
Go to Settings - Player and set Buffer-Output to 16384 .
This happens if you have a large Database.

**Reply by kreativkod** [reply]
What does this do? Does it allocate more memory to database operations or something? My database is 70 000 files.

Is a rebuild of the library necessary afterwards?

**Reply by hjfisch** [reply]
No rebuild necessary. Afterwards you should see all your Albums. Was the same for me. I saw all my Interprets but none of my Albums (over 5000). After changing this setting(b.t.w. it was explaint in this forum before, just do a search) all were shown.

**Reply by kreativkod** [reply]
Thanks! So I raised the buffer to 65536 and that makes it possible to browse genre>classical. But not even raising it to double that makes album browsing possible. I saw in that thread it being mentioned that " in album names would break the script. This could be my case as well because I have over 300 albums with " or ' in the album title. 

Before beginning to retag all those albums, do we know if @rern is addressing this in an existing/forthcoming update? I would think the use case for having " and ' in album titles is very widespread and that the most logical solution would be to tweak the scripts so that they can allow " and ' in album titles.

**Reply by kreativkod** [reply]
It does seem that indexing of albums has been prevented...

`
notremusique:/srv/http/data/mpd  #  ls -la
total 2448
drwxr-xr-x  2 mpd  audio    4096 Jan 11 22:59 .
drwxr-xr-x 12 root root     4096 Dec  4 09:06 ..
-rw-r--r--  1 mpd  audio       0 Jan 11 19:48 album
-rw-r--r--  1 mpd  audio   15439 Jan 11 19:48 albumartist
-rw-r--r--  1 mpd  audio   15328 Jan 11 19:48 artist
-rw-r--r--  1 mpd  audio    6635 Jan 11 19:48 composer
-rw-r--r--  1 mpd  audio    1025 Jan 11 19:48 conductor
-rw-r--r--  1 mpd  audio     192 Jan 11 19:48 counts
-rw-r--r--  1 mpd  audio    7707 Jan 11 19:48 date
-rw-r--r--  1 mpd  audio      67 Jan 11 19:48 genre
-rw-r--r--  1 mpd  audio 2432464 Jan 11 19:48 mpd.db
-rw-r--r--  1 mpd  mpd       758 Jan 11 22:59 mpdstate

`

**Reply by kreativkod** [reply]
epilogue: So after doing a library rebuild the albums also were indexed, without further measures taken. Funny, but good!

**Comment by ceejayemm** [comment]
I am not sure about 320 albums is a big database but I will give it a go.

Chris

**Reply by hjfisch** [reply]
I've tested a lot of mediaplayers over the last years and they are all very sesitive in tagging. Some don't like ö, ü or ä and some won't allow +,“ in the tags. So I do only a minimal tagging with Album Name, Titel Nr., Title,Genre, Date, Interpret and Albuminterpret, where the last 2 should be the same, because otherwise some players could not handle it. The easiest way is to do it is mp3tag. It takes nearly a month and some grey hairs to bring all of my flacs in this form. But now they are all displayed correct in raudio, volumio, cocktail Audio, etc.

**Comment by rern** [comment]
**rAudio** should have no troubles displaying tags as long as they're UTF-8. There's no restrictions for `"`, `'`, symbols and characters with diacritic/accent.
With an album or a track which is displayed correctly, try inserting susceptible characters with `Tag Editor` and report any found problems.
